repository-data/application/src/main/resources/hcm-config/configuration/update/EximImport.groovy

```diff
@@ -29,11 +29,11 @@ import org.onehippo.forge.content.exim.core.*
 import org.onehippo.forge.content.exim.core.impl.*
```

Comment:
Is this just roll back to original Exim i mporter?

Reply:
It is, yes.  `MigratorImporterScript` targets files generated by the Migrator application and that means that it no longer qualifies as a generic importer.

Since we're apparently providing vanilla Exim Exporter, it seemed to make sense to continue providing corresponding vanilla Exim Importer as well so when I introduced `MigratorImporterScript`, I've restored the original Exim Importer script.

Reply:
:+1: